src/Types/Int64Type.php

```diff
@@ -21,8 +22,28 @@ public function convertToDatabaseValue($value)
         return new Int64($value);
     }
 
+    public function convertToPHPValue(mixed $value): ?int
+    {
+        return $value !== null ? (int) $value : null;
+    }
+
     public function closureToMongo(): string
     {
         return '$return = new \MongoDB\BSON\Int64($value);';
     }
+
+    public function closureToPHP(): string
+    {
+        return '$return = (int) $value;';
+    }
+
+    public function diff(mixed $old, mixed $new): int
+    {
+        return (int) $new - $old;
```

Comment:
The `(int)` cast has priority over the `-` operation. This should be unit tested using floats as input values.
```suggestion
        return (int) ($new - $old);
```